frontend/src/scenes/persons/PersonPreview.tsx

```diff
@@ -28,9 +28,18 @@ export function PersonPreview(props: PersonPreviewProps): JSX.Element | null {
         return <Spinner />
     }
 
-    // NOTE: This should pretty much never happen, but it's here just in case
+    // NOTE: This can happen if the Person was deleted or the events associated with the distinct_id had person processing disabled
     if (!person) {
-        return <>Not found</>
+        return (
+            <div className="p-2 max-w-160">
+                <h4>Person not found</h4>
+                <p>
+                    The Person may have been deleted.
+                    <br />
+                    Alternatively, the events for this user may have had Person Profiles disabled.
+                </p>
```

Comment:
```suggestion
                <p>
                    Person profiles are only created when events have `blah` or `thingy` set (link to docs). The events for this user were not thingy events, or the person was deleted.
                </p>
```

although we can't link to docs in tooltips 🙈 

trying to say the default is that person is not found (iiuc) and here's the upsell for you

Reply:
In this one we can - it's an interactive tooltip 😯